light-blockchain/src/chain_store.rs

```diff
@@ -72,17 +72,15 @@ impl ChainStore {
         let hash = chain_info.head.hash();
 
         // We only store in the ChainStore:
-        // - Micro blocks: Headers.
+        // - Micro blocks: Headers and Justification
```

Comment:
For consistency, I'd suggest:
```suggestion
        // - Micro blocks: Headers and justifications.
```